app/components/support_interface/application_navigation_component.rb

```diff
@@ -33,14 +33,25 @@ def sentry_link
     end
 
     def logit_link
-      environment = HostingEnvironment.environment_name
-      candidate_id = @application_form.candidate_id
-      link = "https://kibana.logit.io/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-14d,to:now))&_a=(columns:!(_source),index:'8ac115c0-aac1-11e8-88ea-0383c11b333a',interval:auto,query:(language:kuery,query:'candidate_id:#{candidate_id}+AND+hosting_environment:#{environment}'),sort:!('@timestamp',desc))"
+      link = "https://kibana.logit.io/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15d,to:now))&_a=(columns:!(_source),filters:!(),index:'8ac115c0-aac1-11e8-88ea-0383c11b333c',interval:auto,query:(language:kuery,query:'application:%22#{current_logit_application_name}%22%20AND%20payload.candidate_id:#{@application_form.candidate_id}'),sort:!('@timestamp',desc))"
```

Comment:
was it intentional we changed this to 15 days as opposed to 14 days? Not sure it matters though